File: DDD.clj
(def DDD (read-string (read-line)))
(def mapa
  {61 "Brasilia"
   71 "Salvador"
   11 "Sao Paulo"
   21 "Rio de Janeiro"
   32 "Juiz de Fora"
   19 "Campinas"
   27 "Vitoria"
   31 "Belo Horizonte"})

(def resp (get mapa DDD))

(if (not= resp nil)
  (println resp)
  (println "DDD nao cadastrado"))

File: ddd.hs
ddd :: Integer -> String
ddd n
    | n == 61 = "Brasilia"
    | n == 71 = "Salvador"
    | n == 11 = "Sao Paulo"
    | n == 21 = "Rio de Janeiro"
    | n == 32 = "Juiz de Fora"
    | n == 19 = "Campinas"
    | n == 27 = "Vitoria"
    | n == 31 = "Belo Horizonte"
    | otherwise = "DDD nao cadastrado"

main :: IO ()
main = do
    num' <- getLine
    let num = read num' :: Integer
    putStrLn $ ddd num